app/javascript/components/widgets/twitter/panel.jsx

```diff
@@ -71,13 +92,18 @@ export const TabBar = () => (
 
 const Twitter = ({ tweets }) => {
   const latestTweet = tweets[0];
-  const { createdAt, text, interactions, media, user } = latestTweet;
+  const { status, createdAt, text, interactions, media, user } = latestTweet;
 
   return (
     <PanelWrapper>
+      <RetweetWrapper displayStyle={status === 'retweet' ? 'flex' : 'none'}>
```

Comment:
Nit: prefer this ternary was moved into the styles and you pass `status` as a prop